Themes/default/languages/EmailTemplates.english.php

```diff
@@ -385,7 +385,7 @@
 $txt['paid_subscription_reminder_subject'] = 'Subscription about to expire at {FORUMNAME}';
 $txt['paid_subscription_reminder_body'] = '{REALNAME},
 
-A subscription you are subscribed to at {FORUMNAME} is about to expire. If when you took out the subscription you selected to auto-renew you need take no action, otherwise you may wish to consider subscribing once more. Details are below:
+A subscription you are subscribed to at {FORUMNAME} is about to expire. If you selected to auto-renew, you do not need to take action; otherwise, you may wish to consider subscribing once more. Details are below:
```

Comment:
@Sesquipedalian does this suggestion look good to you?
```suggestion
Your subscription at {FORUMNAME} is about to expire. If you have elected to auto-renew, you do not need to take action; otherwise, you may wish to consider subscribing once more. Details are below:
```

Reply:
Yes, it does.